story3.html

```diff
@@ -1,46 +1,46 @@
 <!DOCTYPE html>
 <html>
 
-<head style= "background-color: #e5ba57">
+<head class="story3Head">
```

Comment:
The `<head></head>` tag in HTML documents is not a visible element. Remove the class.